### DISCUSSION TITLE
CN can't postinstall via `yarn add`

### DISCUSSION THREAD

**Discussion by tiltysola** [coder/code-server#2251] [Q&A] [answered]
error /usr/local/share/.config/yarn/global/node_modules/code-server: Command failed.
Exit code: 35
Command: ./postinstall.sh
Arguments: 
Directory: /usr/local/share/.config/yarn/global/node_modules/code-server
Output:
curl: (35) TCP connection reset by peer

**Comment by code-asher** [comment, marked as answer]
It looks like some kind of network issue. Maybe the cloud agent is failing to download?


https://github.com/cdr/code-server/blob/62735da69466a444561ab9b1115dc7c4d496d455/ci/build/npm-postinstall.sh#L27-L29

We should catch the error so it won't fail the install.

**Reply by tiltysola** [reply]
Yes, I use https_proxy to fixed it! tks
